source/releasenotes/2025-01-15-php-53-54-55-eol.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "PHP 5.3-5.5 will not be available February 25, 2025"
+published_date: "2025-01-15"
+categories: [infrastructure, security, action-required]
+---
+As part of our continued effort to protect and secure customer sites, the Pantheon platform will no longer offer PHP versions 5.3, 5.4, and 5.5 as of February 25, 2025.
+
+PHP 5.5 was declared end-of-life (EOL) by [the PHP Foundation](https://www.php.net/supported-versions.php) on July 21, 2016, more than eight years ago. EOL software does not receive security or feature updates, and could expose sites to attack if any vulnerabilities or exploits are discovered.
+
+We will soon provide LTS security coverage for all available PHP versions on the platform.
+
+**Action required**
+
+Customers with sites using PHP 5.3-5.5 should test and [upgrade the PHP version](https://docs.pantheon.io/guides/php/php-versions) to at least PHP 7.2. If no action is taken, sites using PHP 5.5-5.5 will be auto-upgraded to PHP 5.6 on February 25, 2025. Pantheon [currently recommends](/guides/php#supported-php-versions) at least PHP 8.1 for all production sites.
```

Comment:
5.5-5.5 should be "5.3-5.5" or "5.3 or 5.3". I would recommend using "and" or "or" everywhere.